README.md

```diff
@@ -14,147 +14,38 @@ OpenFunction features but not limited to the following:
 - Provide event management functions for trigger functions.
 - Provide additional functions to manage function versions, ingress management etc.
 
-## ☸ CustomResourceDefinitions
```

Comment:
Instead of deleting the entire CustomResourceDefinitions part, maybe we can keep the components overview picture as part of a section called Components and move the content to a file name `docs/components.md`, then add a link to this file before we add it's content to the website

Reply:
add the link